File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/klaytnDebug/blockchainInspection/DebugStartContractWarmUpTest.java
package opensdk.sdk.apis.klaytnDebug.blockchainInspection;

import opensdk.sdk.apis.constant.UrlConstants;
import opensdk.sdk.models.DebugStartContractWarmUpResponse;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Debug RPC Test")
public class DebugStartContractWarmUpTest {
    private final OpenSDK sdk = new OpenSDK(UrlConstants.SERVER_URL);

    @Test
    @DisplayName("RPC debug_startContractWarmUp")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        String address = "0xfD1d63a152f7A7Ef14bd157C1c73c5bC3239EA5D";

        DebugStartContractWarmUpResponse response = sdk.debug.startContractWarmUp(address).send();
        response.getResult();
    }
}

File: web3rpc/sdk/client/javascript/openapi-test/test/debug/blockchainInspection/DebugStartContractWarmUp.test.js
const OpenSdk = require("@klaytn/web3rpc");
const { expect } = require("@jest/globals");
const { RPC } = require("../../constant");

const sdk = new OpenSdk.DebugApi(new OpenSdk.ApiClient(RPC));

describe('debug_startContractWarmUp API', () => {
    test('should return debug_startContractWarmUp', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data).toBeDefined()
            done();
        };

        const address = "0xfD1d63a152f7A7Ef14bd157C1c73c5bC3239EA5D"

        sdk.startContractWarmUp(address, {}, callbackOne);
    });
});

